docs/api/clojure/ndarray.md

```diff
@@ -0,0 +1,118 @@
+# NDArray API
+
+
+The ndarray api contains tensor operations similar to `numpy.ndarray`. The syntax is also similar, except for some additional calls for dealing with I/O and multiple devices.
+
+Topics:
+
+* [Create NDArray](#create-ndarray)
+* [NDArray Operations](#ndarray-operations)
+* [NDArray API Reference](http://mxnet.incubator.apache.org/api/clojure/docs/org.apache.clojure-mxnet.ndarray.html)
+
+To follow along with this documentation, you can use this namespace to with the needed requires:
+
+```clojure
+(ns docs.ndarray
+  (:require [org.apache.clojure-mxnet.ndarray :as ndarray]
+            [org.apache.clojure-mxnet.context :as context]))
+```
+
+
+## Create NDArray
+
+Create `mxnet.ndarray` as follows:
+
+```clojure
+
+(def a (ndarray/zeros [100 50])) ;;all zero arrray of dimension 100 x 50
+(def b (ndarray/ones [256 32 128 1])) ;; all one array of dimension
+(def c (ndarray/array [1 2 3 4 5 6] [2 3])) ;; array with contents of a shape 2 x 3
+```
+
+There are also ways to convert to a vec or get the shape as an object or vec
```

Comment:
convert what to a vec?
convert NDArray to vec. You can also get the shape (of what? the NDArray) as an object or vec as follows:

Reply:
I'll clarify